vector/src/main/java/im/vector/app/features/spaces/SpaceSettingsMenuBottomSheet.kt

```diff
@@ -138,8 +148,38 @@ class SpaceSettingsMenuBottomSheet : VectorBaseBottomSheetDialogFragment<BottomS
         }
 
         views.leaveSpace.views.bottomSheetActionClickableZone.debouncedClicks {
+            val spaceSummary = activeSessionHolder.getSafeActiveSession()?.getRoomSummary(spaceArgs.spaceId)
+                    ?: return@debouncedClicks
+            val warningMessage: CharSequence = if (spaceSummary.otherMemberIds.isEmpty()) {
+                span {
+                    +getString(R.string.space_leave_prompt_msg)
+                    +"\n"
+                    span(getString(R.string.space_leave_prompt_msg_only_you)) {
+                        textColor = ContextCompat.getColor(requireContext(), R.color.riotx_destructive_accent)
+                    }
+                }
+            } else if (isAdmin) {
+                span {
+                    +getString(R.string.space_leave_prompt_msg)
+                    +"\n"
+                    span(getString(R.string.space_leave_prompt_msg_as_admin)) {
```

Comment:
We could check if we are the latest admin in the space, no?

Reply:
Added